imqsauth/perms.go

```diff
@@ -1,381 +1,767 @@
-package imqsauth

```

Comment:
It's strange that this diff is so big. Did you perhaps format it with gofmt, after using the script to generate it? That's just a guess.

Reply:
It must be a difference between Linux and Windows cause I ran the script again with no difference in the output